pageserver/src/task_mgr.rs

```diff
@@ -217,9 +217,10 @@ pageserver_runtime!(COMPUTE_REQUEST_RUNTIME, "compute request worker");
 pageserver_runtime!(MGMT_REQUEST_RUNTIME, "mgmt request worker");
 pageserver_runtime!(WALRECEIVER_RUNTIME, "walreceiver worker");
 pageserver_runtime!(BACKGROUND_RUNTIME, "background op worker");
+pageserver_runtime!(OTEL_RUNTIME, "open telemetry worker");
```

Comment:
https://github.com/neondatabase/neon/pull/11139#discussion_r1988805639

> NB there are no users of init_tracing_without_runtime in the codebase. I would prefer to see this killed, having multiple runtimes is something we want to move away from.

Reply:
Decided out-of-band on using the compute request runtime for now.
Done in https://github.com/neondatabase/neon/pull/11140/commits/5e58bfe04bdf8654e81cfa0a4036840b09e9c0f1.

Reply:
Comment on why COMPUTE_REQUEST_RUNTIME would have been nice